SCons/Tool/gxx.py

```diff
@@ -67,6 +224,17 @@ def generate(env) -> None:
     env['CCDEPFLAGS'] = '-MMD -MF ${TARGET}.d'
     env["NINJA_DEPFILE_PARSE_FORMAT"] = 'gcc'
 
+    env['__CXXMODULEINIT__'] = init_mapper
+    if sys.platform != "win32":
+        env['__GCCMAPPERMODE__'] = "domain_socket"
+        env['__GCCMAPPERFLAGS__'] = "=$CXXMODULEPATH/socket"
```

Comment:
Remove the extra `=` here. It ends up failing:

```
/usr/include/c++/13/map: error: failed opening mapper '=cxx-modules/socket'
```

Reply:
https://gcc.gnu.org/onlinedocs/gcc/gcc-command-options/c%2B%2B-modules.html#module-mapper <- `=` introduces a socket based mapper.

Reply:
Wow, you're right. Weird syntax :)

Looking more closely it was failing because the mountpoint didn't allow a domain socket to be created.

Reply:
Are you using some unusual os?
FTR I usually run it like this: 
```
~/vcs-checkouts/scons/test/CXX/CXX-modules-fixture [cxx20-gcc|✚ 1…7] $ python ../../../scripts/scons.py toolset=g++
scons: Reading SConscript files ...
scons: done reading SConscript files.
scons: Building targets ...
g++ -o test.o -c -fmodules-ts -fmodule-mapper==cxx-scons-modules/socket?test.cpp -std=c++20 test.cpp
g++ -o itest.o -c -fmodules-ts -fmodule-mapper==cxx-scons-modules/socket?itest.cpp -std=c++20 itest.cpp
g++ -x c++-system-header -c -fmodules-ts -fmodule-mapper==cxx-scons-modules/socket?@system-header/iostream -std=c++20 iostream
g++ -o main.o -c -fmodules-ts -fmodule-mapper==cxx-scons-modules/socket?main.cpp -std=c++20 main.cpp
g++ -o scons-module-test itest.o test.o main.o
scons: done building targets.
```

Reply:
Yeah, it's a build container with a bind mount from a Mac. Just had to move the module directory to /var.